.eslintignore

```diff
@@ -1,2 +1,3 @@
 extension/
-external/
\ No newline at end of file
+external/
+src/mousetrap.min.js
```

Comment:
For any new deps, please use NPM and install via `yarn add $PACKAGE_NAME` rather than directly including their JS in the codebase